### ISSUE TITLE
Allow user click link in read only mode

### ISSUE THREAD

**Issue by zpz1982** [sstur/react-rte#130]
I use read only mode to display content as normal html. In this case, the link should be clickable.

**Comment by ZeroCho** [comment]
Well, I think it's not a good idea. react-rte is an editor not a html viewer.

**Comment by sstur** [comment]
@ZeroCho You're right, it might not be a great idea, but I can understand why you might want that @zpz1982. I'd be open to a PR for this feature.

**Comment by zpz1982** [comment]
Is there a better way to display content just like what user saw in the editor? Or can you provide a CSS file like github-markdown-css does?
